plugins/usbdmx/README.md

```diff
@@ -5,10 +5,12 @@ This plugin supports various USB DMX devices including:
 
 * Anyma uDMX
 * AVLdiy D512
+* Digital Enlightenment
```

Comment:
I think this should be "Digital Enlightenment USB-DMX", I think Digital Enlightenment is the company/group.

Reply:
Done in 30577e3cc69a8acf161f5f1e6dd6a8e617d9b704